



<?xml version="1.0"?>
<sdf version="1.4">
  <model name="tissue_box">
    <link name="link_1">
      <pose>0 0 0.05 0 0 0</pose>
      <gravity>true</gravity>
      <inertial>
        <pose>0 0 0 0 0 0</pose>
        <inertia>
          <ixx>0.16666666666666669</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.16666666666666669</iyy>
          <iyz>0</iyz>
          <izz>0.16666666666666669</izz>
        </inertia>
        <mass>100</mass>
      </inertial>
      <visual name="visual">
        <cast_shadows>0</cast_shadows>
        <geometry>
          <box>
            <size>0.1 0.1 0.1</size>
          </box>
        </geometry>
      </visual>
      <collision name="box_collision">
        <geometry>
          <box>
            <size>0.1 0.1 0.1</size>
          </box>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>0.1</mu>
              <mu2>0.1</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
    </link>
    <static>true</static>
  </model>
</sdf>

